### DISCUSSION TITLE
Release candidate 3.20.0rc0 Testing

### DISCUSSION THREAD

**Discussion by sotorrio1** [CCSI-Toolset/FOQUS#1188] [Show and tell]
I just finished testing the release candidate (3.20.0rc0) on both Mac and Windows. Other than the typos found in the OUU tutorials (reported in issue #1186 and fixed in PR #1187) I haven't found any issues and I was able to run all the tutorials successfully. More details below:

- _Mac OS Ventura 13.6.3_
    - **UQ:** five [tutorials](https://foqus.readthedocs.io/en/stable/chapt_uq/tutorial/index.html) from documentation.
    - **OUU**: seven [tutorials](https://foqus.readthedocs.io/en/stable/chapt_ouu/tutorial.html) from documentation.
    - **SDoE**: six [examples](https://foqus.readthedocs.io/en/stable/chapt_sdoe/examples.html) from documentation (three USF, 2 NUSF, 1 IRSF).
    - **ODoE**: two [tutorials](https://foqus.readthedocs.io/en/stable/chapt_odoe/tutorials.html) from documentation.

- _Windows 10 Enterprise_
    - **UQ:** five [tutorials](https://foqus.readthedocs.io/en/stable/chapt_uq/tutorial/index.html) from documentation.
    - **OUU**: seven [tutorials](https://foqus.readthedocs.io/en/stable/chapt_ouu/tutorial.html) from documentation.
    - **SDoE**: six [examples](https://foqus.readthedocs.io/en/stable/chapt_sdoe/examples.html) from documentation (three USF, 2 NUSF, 1 IRSF).
    - **ODoE**: two [tutorials](https://foqus.readthedocs.io/en/stable/chapt_odoe/tutorials.html) from documentation.

**Comment by lbianchi-lbl** [comment]
Thank you so much @sotorrio1, this is wonderful!

**Comment by ksbeattie** [comment]
Let's move this into the new DevOps repo.
